db/builder.cc

```diff
@@ -145,10 +145,16 @@ Status BuildTable(
       file->SetIOPriority(io_priority);
       file->SetWriteLifeTimeHint(write_hint);
 
+      bool data_verification = false;
```

Comment:
Should this be a method somewhere?  There is almost the same code block in many locations